lib/browser/api/touch-bar.js

```diff
@@ -118,6 +118,9 @@ class TouchBar extends EventEmitter {
       window.removeListener('closed', removeListeners)
       window._touchBar = null
       delete this.windowListeners[id]
+      for (const item of this.ordereredItems) {
```

Comment:
'ordereredItems'? not new to this PR, but... :roll_eyes: :smile:

Reply:
`items` is a map of id --> item
`orderedItems` is an array giving you the left->right order of those items containing references to the objects in the `items` map

Reply:
Right, I get it. Just amused by the spelling `ordereredItems` :2nd_place_medal:

Reply:
Oh, wow, ok.  That's a typo I never noticed :laughing: